client/consensus/babe/src/authorship.rs

```diff
@@ -114,6 +114,7 @@ pub(super) fn secondary_slot_author(
 		return None;
 	}
 
+	dbg!((randomness, slot_number).using_encoded(blake2_256));
```

Comment:
is this to be removed ?

Reply:
yes I was debugging a strange situation which you can now see in the CI: https://gitlab.parity.io/parity/substrate/-/jobs/774551

At some point, I introduced a panic somewhere apparently, if you run a `--dev` chain with this branch now it panics at the first block. Backtrace was opaque to me. git bisect revealed that issue was introduced in`4732854748fe61cfc7020b17860a4fadccc3f0b0`, but that's an empty commit. Looking into it.